dataplane/dregress.cpp

```diff
@@ -4,19 +4,45 @@
 #include "common/fallback.h"
 
 #include "checksum.h"
-#include "controlplane.h"
 #include "dataplane.h"
 #include "dregress.h"
 #include "metadata.h"
-#include "worker.h"
+#include "slow_worker.h"
+
+dregress_t::dregress_t(dataplane::SlowWorker* slow, cDataPlane* dataplane, uint32_t gc_step) :
+        slow_worker_{slow},
+        dataplane{dataplane},
+        stats{},
+        connections{new dregress::ConnTable},
```

Comment:
That's bad. I suggest avoiding raw pointers alongside allocating memory with `new` in constructor and calling `delete` in destructor. That's what `std::unique_ptr` does.
Here it will be `connections_(std::make_unique<dregress::ConnTable>())`

Reply:
I agree, but this doesn't relate to the theme of commit. This should be done in separate commit.